### PR TITLE
Harmonize receipts

### LINKED ISSUE #837: Harmonize receipts and reports
This issue is about to provide an harmonized format for receipts and reports : 

Proposals: 

A. FOR RECEIPTS

1. Receipt for invoice : 

facture

2. Receipt for cash payment : 

paiement

3. Receipt for voucher : 

voucher

B. FOR REGISTRY 

registry

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR is about harmonizing receipts for : 
- Patient invoice
- Cash payment 
- Voucher 

close #837

### CODE REVIEW

**Comment by mbayopanda** [comment]
@sfount could you review this PR

File: server/controllers/finance/reports/cash/receipt.handlebars [hunk 1]
```diff
@@ -1,41 +1,50 @@
 {{> head title="CASH.TITLE" }}
 
-<div class="container" style="font-size: 0.8em;">
+<div class="container" style="font-size: 0.9em;">
```
Comment (sfount) [inline server/controllers/finance/reports/cash/receipt.handlebars:3]: In the future we should clearly define font family, size and style uniformly across all documents.

File: server/controllers/finance/reports/invoices/receipt.handlebars [hunk 2]
```diff
@@ -153,7 +134,9 @@
     </div>
   </div>
 
-  <h4>{{translate 'FORM.LABELS.SIGNATURE'}}</h4>
+  <h4>
+    <u>{{translate 'FORM.LABELS.SIGNATURE'}}s</u>
```
Comment (sfount) [inline server/controllers/finance/reports/invoices/receipt.handlebars:138]: Is this 's' intentional?

**Review by sfount — APPROVED** [review]
These receipts LGTM. Excellent first step towards having standardised documents.

**Pull request merged**